### PR TITLE
Feature: add "amount" for "burn" smart contract method

### LINKED ISSUE #2321: Feature: add "amount" for "burn" smart contract method

### PR DESCRIPTION
**Pull request by YarikRevich**

### CODE REVIEW

File: code/go/0chain.net/smartcontract/dbs/goose/migrations/001_events_db.sql [hunk 1]
```diff
@@ -1032,6 +1032,7 @@ CREATE TABLE public.burn_tickets (
                               updated_at timestamp with time zone,
                               ethereum_address text,
                               hash text,
+                              amount bigint,
```
Comment (dabasov) [inline code/go/0chain.net/smartcontract/dbs/goose/migrations/001_events_db.sql]: add it to the different migration as an alter table

**Review by dabasov — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/smartcontract/dbs/goose/migrations/001_events_db.sql (hunk 1 above)
Comment (YarikRevich) [reply code/go/0chain.net/smartcontract/dbs/goose/migrations/001_events_db.sql]: Done

**Review by dabasov — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | TRUE  |
| system-tests  | master  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | feature/amount_field  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Pull request merged**